src/utility/Translation.ts

```diff
@@ -74,7 +74,7 @@ export class Translation {
   static CONNECTION_ADD_CONNECTION_BEFORE = `Please add first a connection for this feature.`;
   static CONNECTION_ADD_SERVER_BEFORE = `Please add first a Server to create a Connection.`;
   static CONNECTION_LIST_TABLEHEADERS = ['Connection Name', 'Description'];
-  static CONNECTION_RENDER_METAMODEL_LABEL = 'add Metamodelcontainer';
+  static CONNECTION_RENDER_METAMODEL_LABEL = 'generate code';
```

Comment:
For readability it might be a good idea to change CONNECTION_RENDER_METAMODEL_LABEL to CONNECTION_GENERATE_CODE as well.